### PR TITLE
Code formatting and debug logs

### PR DESCRIPTION
**Pull request by cristianonicolai**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/README.md)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] [KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by cristianonicolai** [comment]
no actual changes just a bung of code reformat on some classes while doing investigation, plus debug logs for jobs.

File: jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java [hunk 1]
```diff
@@ -77,219 +77,211 @@
...
+        result.addClassLoader(Thread.currentThread().getContextClassLoader());
+        return result;
+    }
+
     public ProcessInstance startProcess(final String processId) {
         return startProcess(processId, null);
     }
 
     public ProcessInstance startProcess(String processId,
                                         Map<String, Object> parameters) {
```
Comment (MarianMacik) [inline jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java:167]: What is our current code style? Somewhere it is one param per line, somewhere it is all on one line.

Comment (cristianonicolai) [reply jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java:167]: this is a leftover from a previous style, we used to enforce one parameter per line.

Comment (MarianMacik) [reply jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java:167]: thanks

**Pull request merged**